driver/portfolio_runner.py

```diff
@@ -87,7 +87,7 @@ def compute_run_time(timeout, configs, pos):
           pos, relative_time, remaining_relative_time))
     # For the last config we have relative_time == remaining_relative_time, so
     # we use all of the remaining time at the end.
-    return remaining_time * relative_time / remaining_relative_time
+    return int(remaining_time * relative_time / remaining_relative_time)
```

Comment:
I think it would be helpful to output the resulting time limit in the print() function directly above.

Reply:
The satisficing part of the code skips configs with time limits <= 0, but the optimal one doesn't. Do you want to add
```
if run_time <= 0:
    return
```
to `run_opt`?